packages/block-scripts/.eslintrc.cjs

```diff
@@ -3,6 +3,8 @@ module.exports = {
   ...require("@local/eslint-config/generate-workspace-config.cjs")(__dirname),
   rules: {
     "no-console": "off",
-    "import/extensions": ["error", "always"],
+    "import/extensions": ["error", "ignorePackages"],
+    "import/named": "off", // Need to investigate this (Maximum call stack size exceeded Occurred while linting ... Rule: "import/named")
+    "import/no-cycle": "off", // Need to investigate this (Maximum call stack size exceeded Occurred while linting ... Rule: "import/no-cycle")
```

Comment:
These two rules were hanging VSCode, so I disabled them for now. This may be to do with some recent update in `eslint-plugin-import`. We can try the rules again in a few weeks and see what happens.